docs/snippets/dydx-quickstart-guide.mdx

```diff
@@ -1,7 +1,155 @@
 <AccordionGroup>
+    <Accordion title="dYdX v7">
+
+        **The required version for Connect with dYdX v7 is `v1.0.13`.**
+
+        First, please ensure you've received your **API keys** for the relevant decentralized provider nodes. If you have not received API keys, please reach out to the provider team in the relevant communication channels.
+
+        Next, place your API keys under their corresponding URLs in the following file and save it to your system. **Keep the file path handy** as we will pass it into a flag when running Connect.
+
+        We will also supply an edited configuration for the `dydx_migration_api` which facilitates graceful migration from dydx's `x/prices` to `x/marketmap`.
+
+        For the `dydx_migration_api` provider, **make sure to fill in the URL for the REST endpoint and gRPC endpoint of your node** (in that order). The migration API will _not_ work unless the REST API endpoint is the **first** endpoint in the endpoints list.
+
+        ```json oracle.json
+        {
+            "providers": {
+            "dydx_migration_api": {
+            "api": {
+            "endpoints": [
+        {
+            "url": "http://<YOUR NODE REST API ENDPOINT>"
+        },
+        {
+            "url": "<YOUR NODE GRPC ENDPOINT>:<YOUR NODE GRPC PORT>"
+        }
+            ]
+        }
+        },
```

Comment:
given that v6 uses x/marketmap, i don't think users going from v6 -> v7 need to do this migration stuff anymore

Reply:
Yeah, I just wanted to keep the doc the same as people were running for dydx to keep the change that validators see as minimal as possible